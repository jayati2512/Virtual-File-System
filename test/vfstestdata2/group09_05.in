createvfs demo 1024
mountvfs demo1
mountvfs demo
makedir /home/dir1/ newdir1
makedir /home/dir1/ newdir2
makedir /home/dir1/dir3/ newdir3
listdir /home/dir1/ 0 /tmp/grp9_listdir0
addfile /home/dir1/dir3/ newfile1.txt /usr/include/syscall.h
listdir /home/dir1/ 1 /tmp/grp9_listdir1
unmount demo
createvfs demo 100
mountvfs demo
listfile /home/dir1/dir3/newfile1 /tmp/listfile_newfile1
addfile /home/dir1/ newfile2.txt /usr/include/stab.h
listfile /home/dir1/newfile2 /tmp/listfile_newfile2
makedir /home
makedir /home/dir1/ newdir3
movedir /home/dir1/dir3/newdir3 /home/dir1/
listdir /home/dir1/ 10 /tmp/grp9_listdir10
deletedir /home/dir1/
copyfile /home/dir1/dir3/ /home/dir1/dir3/newfile3
addfile /home/dir1/ newfile4.txt /usr/include/syscall.h
addfile /home/dir1/ /usr/include/stab.h
addfile /home/dir1/ newfile6.txt /usr/include/syslog.h
addfile /home/dir1/ newfile7.txt /usr/include/syscall.h
addfile /home/dir1/ newfile8.txt /usr/include/ustat.h
addfile /home/dir1/ newfile9.txt /usr/include/stab.h
updatefile /home/dir1/newfile6 /usr/include/ustat.h
removefile /home/dir1/newfile1
unmountvfs demo
