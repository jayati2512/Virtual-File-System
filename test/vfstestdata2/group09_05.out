createvfs_SUCCESS
mountvfs_FAILURE CANNOT_READ_FROM_FILE
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
addfile_SUCCESS
listdir_SUCCESS
Ignoring invalid command unmount
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
addfile_SUCCESS
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
movedir_SUCCESS
listdir_FAILURE INVALID_FLAG
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
copyfile_FAILURE CANNOT_FIND_DESTINATIONPATH
addfile_SUCCESS
addfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
updatefile_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_FILE
unmountvfs_SUCCESS
